.github/workflows/build-package-docs.yaml

```
@@ -120,3 +124,76 @@ jobs:
         name: package-docs-build
         path: build-directory/docs/docsite/ansible-package-docs-html-*.tar.gz
         retention-days: 7
+
+  deploy-package-docs:
+    needs: build-package-docs
+    runs-on: ubuntu-latest
+    environment:  # TODO create environment and specify it here
+    env:
+      dest-repo: owner/repo-name  # TODO create repo and specify here
+      dest-branch: main  # TODO specify repo branch here
+      user-email: username@example.com  # TODO specify email for commit
+      user-name: username  # TODO specify username for commit
+    steps:
+    - name: Download the build archive
+      uses: actions/download-artifact@v4
+      with:
+        name: package-docs-build
+
+    - name: Extract the tarball
+      run: |
+        mkdir -p extracted-docs
```

Make this verbose and strict (since there's more than one command here):
```suggestion
        set -xeEuo pipefail
        mkdir -pv extracted-docs
```